### ISSUE TITLE
pgapp timeout issue

### ISSUE THREAD

**Issue by jonathan-augenstine** [epgsql/pgapp#19]
We are having an issue with utilizing the pgapp on some larger queries.  We are receiving timeout error reports.  It works fine for smaller queries and the queries actually succeed (they are inserts to be exact), but a timeout is being reported.  We are trying to understand how/where to set the timeout to a value longer than the default.

**Comment by davidw** [comment]
Hi,

What kind of error messages are you getting? Does it show where the timeout is happening?

**Comment by jonathan-augenstine** [comment]
{error, timeout} is the error we are getting.  The timeout occurs when we send the query via equery and pooling we encounter the timeout.  When we run the query without pooling, just calling equery after connect, then it succeeds without any timeout errors.  We have tried digging in to change the timeout, but without success.  We have tried via both config file and setting it dynamically before initiating the query.
